GoogleDataTransport/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # v1.1.2
 - Add initial support for iOS 13.
 - Add initial support for Catalyst.
+- Backgrounding in GDTStorage is fixed. (#3627)
```

Comment:
Use issue numbers when available instead of PR

Reply:
Done and merging.